### ISSUE TITLE
Add focus indicators to nav items and links

### ISSUE THREAD

**Issue by addiebarron** [phillycommunitywireless/phillycommunitywireless#42]
We need better focus indicators for our nav items and links. We're trying to use the [Tachyons CSS library ](https://tachyons.io) for styles as much as possible, but it doesn't have a lot of options for focus, so this will need to be added in a CSS file. Standardizing our "extra CSS" location would be a useful byproduct of this!

[Deque's guide on focus indicators](https://www.deque.com/blog/give-site-focus-tips-designing-usable-focus-indicators) is a great place to start for preparation.

**Comment by addiebarron** [comment]
For an example, head to any of these websites and press `Tab` to move focus around the page.

https://addieis.online/
https://www.a11yproject.com/
https://www.w3.org/WAI/standards-guidelines/wcag/
https://www.deque.com/

**Comment by qbatten** [comment]
Hi! I'm interested in helping out with this. Haven't done much front-end/accessibility stuff so the Deque guide was super helpful. Here's what I'm thinking...

Seems like there may be a few sub-tasks that need to happen in order for this to be complete:
1. Writing some CSS that makes the focus indicators clearer, bigger, more compliant (higher contrast against background). Specifically, probably a few rules targeting `a:focus` and maybe a few other tags too.
2. Adding `tabindex="0"` to buttons & components on the page, so that cycling focus w tab allows users to access the important buttons/links. (Right now, using Firefox 103, I don't see many tab-focusable elements, I think it's just the main div and then some components in the embedded Youtube videos.)
3. Coming up with a standard way to organize extra CSS bits and bobs (like the code that'll be used to make point 1 happen).

Questions:
- Does the above seem roughly correct? Anything I seem to be missing or that I should know before proceeding?
- The approach that comes to mind for item 2, (adding tabindex attrs to relevant html tags), is going into your theme fork and manually adding `tabindex="0"` to things like e.g. [this line](https://github.com/theNewDynamic/gohugo-theme-ananke/blob/470ea40982f5036554819253c3ac6ed4a34193f4/layouts/partials/site-navigation.html#L15), which creates a nav item.
  - Pros: This should solve it, andd I can't think of an alternative strategy at the moment.
  - Cons: It could maybe result in annoying merges when pulling in new commits from the original theme, since we're going in and making a bunch of small changes in a ton of different files. 
  - What do you think of that approach? How concerning is the annoying-merge bit to you? Any other pitfalls or alternative approaches you'd like to raise?
- For item 3, organizing custom CSS snippets, I see [this section](https://github.com/theNewDynamic/gohugo-theme-ananke/tree/470ea40982f5036554819253c3ac6ed4a34193f4#css) in Ananke's README, and it looks like that strategy has been followed with [at least one file](https://github.com/phillycommunitywireless/phillycommunitywireless/blob/master/assets/pcw-hugo-theme/css/custom.css). I believe we could just keep adding files there and name/organize them however we want, as long as any new files also get registered in config.
  - Does that seem like a good strategy to keep expanding on? Does that somewhat solve the "standardizing extra CSS locations" thing or am I not quite understanding the problem-to-be-solved?

**Comment by addiebarron** [comment]
@qbatten hey!! big thanks for taking this stuff on! your roadmap sounds great. I think this won't necessarily be a lot of work but it should be well considered beforehand, which it sounds like you're doing and then some.

 it's been a little while since I worked on this but I can do my best to answer those questions:
 1. Yes!
 2. Realistically I think we don't need to pull any more changes in from the original theme. We've already diverged quite a lot, and I even think we could pare down our theme since we're not using many of Ananke's more bloggy features. So no need to worry about that. I'm not too well-educated on the specifics of focus accessibility but as long as you're doing your research and testing I'm sure it'll work out.
 3. I think adding to `custom.css` is totally fine for now. I guess what I meant is that eventually we could add those files into the actual theme, but functionally it works out the same (since the theme is unique to this site).

**Comment by qbatten** [comment]
Awesome, will get started on it then!

**Comment by hawc2** [comment]
I just noticed @qbatten's pull request #70, @addiebarron can you take a look?

**Comment by hawc2** [comment]
I merged the pull request and all seems well. I'll be closing this soon if that's cool with everyone. Big thanks to @qbatten!!
